content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -121,10 +121,10 @@ gcloud.compute.zone | The zone to use for zonal resources; must be in gcloud.com
 
   * Kubeflow pipelines currently doesn't work with regional deployments see [kubeflow/gcp-blueprints#6](https://github.com/kubeflow/gcp-blueprints/issues/6)
 
-* The **Makefile** contains a rule `set-values` with appropriate `kpt cfg` commands to set the values
+* The **Makefile** at `./kubeflow/Makefile` contains a rule `set-values` with appropriate `kpt cfg` commands to set the values
```

Comment:
What is the context of `./kubeflow/Makefile`? Is it `${KFDIR}/kubeflow/Makefile` ? If so, consider changing to `${KFDIR}/kubeflow/Makefile`, otherwise specify the directory that this is under.

This should apply to the other instances of `./kubeflow/Makefile`.